addons/mail/static/src/messaging/entity/composer/composer.js

```diff
@@ -203,23 +286,34 @@ function ComposerFactory({ Entity }) {
         }
 
         /**
+         * @param {string} mentionKeyword
          * @private
-         * @param {string} content html content
-         * @returns {integer[]} list of mentioned partner Ids (not duplicate)
          */
-        _getMentionedPartnerIdsFromHtml(content) {
-            const parser = new window.DOMParser();
-            const node = parser.parseFromString(content, 'text/html');
-            const mentions = [...node.querySelectorAll('.o_mention')];
-            const allPartnerIds = mentions
-                .filter(mention =>
-                    (
-                        mention.dataset.oeModel === 'res.partner' &&
-                        !isNaN(Number(mention.dataset.oeId))
-                    )
-                )
-                .map(mention => Number(mention.dataset.oeId));
-            return [...new Set(allPartnerIds)];
+        async _getSuggestedPartners(mentionKeyword) {
+            const mentions = await this.env.rpc({
+                model: 'res.partner',
+                method: 'get_mention_suggestions',
+                kwargs: {
+                    limit: 3,
+                    search: mentionKeyword,
+                },
+            }, { shadow: true });
+
+            const suggestedPartners = mentions[0];
```

Comment:
Results from mentions[1] and separator are currently ignored on purpose? If so, should create a task and add comment with task-id here.

Reply:
Yes, you should handle `mentions[1]` too here.

Reply:
Ok, if the change is not trivial I prefer that we merge without it and just add a fixme with a new task-id, since it doesn't prevent the rest from working.